internal/utils/hashing_test.go

```diff
@@ -0,0 +1,38 @@
+package utils
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestShouldHashString(t *testing.T) {
+	input := "input"
+
+	sum := HashSHA256FromString(input)
+
+	assert.Equal(t, "c96c6d5be8d08a12e7b5cdc1b207fa6b2430974c86803d8891675e76fd992c20", sum)
+}
+
+func TestShouldHashPath(t *testing.T) {
+	dir, err := ioutil.TempDir("", "authelia-hashing")
+	assert.NoError(t, err)
+
+	err = os.WriteFile(filepath.Join(dir, "myfile"), []byte("output\n"), 0600)
+	assert.NoError(t, err)
+
+	sum, err := HashSHA256FromPath(filepath.Join(dir, "myfile"))
+	assert.NoError(t, err)
+
+	assert.Equal(t, "9aff6ba4b042b9d09991a9fbf8c80ddbd2a9c433638339cd831bed955e39f106", sum)
```

Comment:
also check the hash of two different files differ.

Reply:
Done